tests/scripts/stock/soft_delete_stock_test.py

```diff
@@ -0,0 +1,103 @@
+from unittest.mock import patch
+
+import pytest
+
+from pcapi.model_creators.generic_creators import create_booking
+from pcapi.model_creators.generic_creators import create_offerer
+from pcapi.model_creators.generic_creators import create_payment
+from pcapi.model_creators.generic_creators import create_stock
+from pcapi.model_creators.generic_creators import create_user
+from pcapi.repository import repository
+from pcapi.scripts.stock.soft_delete_stock import soft_delete_stock
+
+
+class SoftDeleteStockTest:
+    @patch("pcapi.scripts.stock.soft_delete_stock.repository")
+    @pytest.mark.usefixtures("db_session")
+    def should_return_ko_if_at_least_one_booking_is_used(self, mock_repository, app):
+        # Given
+        user = create_user()
+        stock = create_stock(price=0)
+        booking = create_booking(is_used=True, stock=stock, user=user)
+
+        repository.save(booking)
+
+        # When
+        soft_delete_stock(stock.id)
+
+        # Then
+        assert mock_repository.save.call_count == 0
```

Comment:
Je ne suis pas très fan du mock. Je pense qu'il est préférable (plus explicite, moins sujet à bug) de tester directement si l'object Stock a été soft-supprimé (sic !) ou pas.

```suggestion
        assert not booking.stock.isSoftDeleted
```

idem dans les autres tests plus bas.